itest/lnd_watchtower_test.go

```diff
@@ -0,0 +1,162 @@
+package itest
+
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/funding"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/wtclientrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/lightningnetwork/lnd/lntest/node"
+	"github.com/lightningnetwork/lnd/lntest/wait"
+	"github.com/stretchr/testify/require"
+)
+
+// testWatchtowerSessionManagement tests that session deletion is done
+// correctly.
+func testWatchtowerSessionManagement(ht *lntest.HarnessTest) {
+	const (
+		chanAmt           = funding.MaxBtcFundingAmount
+		paymentAmt        = 100
```

Comment:
you may wanna increase this value to `10_000` to avoid the error,
```
2023-03-17 19:33:44.755 [DBG] SWPR: 1 negative yield inputs not added to input set: f8719e811940e354a9fb964430fd9236bfcb9bc663e855fd1b8fd6159c80440e:0 (CommitmentNoDelayTweakless)
```
guess sth is up with our sweeper🧐

Reply:
cool - managed to get things working by just mining a few blocks at the end of the test to give both sweepers a chance to sweep